src/plugin/duration/index.js

```diff
@@ -25,6 +25,7 @@ const wrapper = (input, instance, unit) =>
   new Duration(input, unit, instance.$l) // eslint-disable-line no-use-before-define
 
 const prettyUnit = unit => `${$u.p(unit)}s`
+const roundNumber = number => number < 0 ? Math.ceil(number) : Math.floor(number)
```

Comment:
This could be replaced with `Math.abs(Math.trunc(number)) || 0`, which is shorter.

Reply:
seems not supported in IE :-(